app/views/support/PlanOps.scala

```diff
@@ -93,16 +81,24 @@ object PlanOps {
         case Delivery => "homedelivery@theguardian.com"
         case Voucher => "vouchersubs@theguardian.com"
         case Product.Digipack => "digitalpack@theguardian.com"
-        case _:Product.Weekly => "gwsubs@theguardian.com"
+        case _: Product.Weekly => "gwsubs@theguardian.com"
         case _ => "subscriptions@theguardian.com"
       }
 
     def faqHref: String =
       product match {
-        case _:Product.Weekly => "https://www.theguardian.com/help/2012/jan/19/guardian-weekly-faqs"
+        case _: Product.Weekly => "https://www.theguardian.com/help/2012/jan/19/guardian-weekly-faqs"
         case _ => "https://www.theguardian.com/subscriber-direct/subscription-frequently-asked-questions"
       }
 
+    def productType: String = product match {
+      case Delivery => "Home Delivery"
+      case Voucher => "Voucher Book"
+      case Product.Digipack => "Digital Pack"
+      case _: Product.Weekly => "Guardian Weekly"
+      case _ => "Unknown"
```

Comment:
I just moved this code from plan to here as it was..
I don't know if it is safe to change that, I'll have to check the js code that uses this.